src/Logging/Logging/src/Microsoft.Extensions.Logging.csproj

```diff
@@ -10,7 +10,7 @@
     <Compile Include="../../shared/*.cs" />
 
     <Reference Include="Microsoft.Extensions.Configuration.Binder" />
-    <Reference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
+    <Reference Include="Microsoft.Extensions.DependencyInjection" />
```

Comment:
It seems that this change caused [these failures](https://dnceng.visualstudio.com/public/_build/results?buildId=57771&view=logs&jobId=f0f0520d-467b-5d1d-a95d-4e3a2581fe8a&taskId=859aa418-c131-516f-eebe-b28b785cce53&lineStart=39&lineEnd=42&colStart=1&colEnd=234) in my PR build. It actually [showed up in your PR check too](https://dnceng.visualstudio.com/public/_build/results?buildId=57675&view=logs&jobId=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&taskId=3d94a178-cdfb-5c90-a971-5deeccbbe043&lineStart=33&lineEnd=34&colStart=1&colEnd=222), but as a warning instead of an error.

@natemcmaster 1. Is the proper solution here (assuming it is an approved breaking change) to [edit this line](https://github.com/aspnet/Extensions/blob/82ad9d6edda284321a10e633cd2863852acf34fe/eng/Baseline.Designer.props#L386)? 2. Should this kind of change be a warning or an error?

Reply:
@ryanbrandenburg does your PR invoke build with `--warnaserror`?

Reply:
Yes, that's what [Arcade defaults to](https://github.com/dotnet/arcade/blob/d7c9d455f3de1edd96618a1e5ab6a668ad138e02/eng/common/tools.ps1#L27). We can certainly override that anywhere (or everywhere), but I thought I'd ask since this seems like the kind of thing we might want to fail a build (even if it's just so we can document that this was an intended break).